ford/FFE.json

```
@@ -0,0 +1,92 @@
+{
+  "init_commands": {
+    "command": ["ATZ", "ATD", "ATE0", "ATS0", "ATAL", "ATSP6"]
+  },
+  "obd_protocol": "7",
+  "data_commands": {
+    "command": [
+      "ATSH7E2",
+      "224850",
+      "224851",
+      "224845"
+      "22DD05",
+      "221E12",
+      "ATSH7E4",
+      "224800",
+      "22480B",
+      "22D115"
+    ]
+  },
+  "current": {
+    "equation": "((signed(A)*256)+B)*0.02",
+    "minValue": "-500",
+    "maxValue": "300",
+    "type": "Number",
+    "command": "22480B",
+    "ecu": "7E4"
+  },
+  "voltage": {
+    "equation": "((A<<8)+B)*0.01",
+    "minValue": "200",
+    "maxValue": "400",
+    "type": "Number",
+    "command": "224850",
+    "ecu": "7E2"
+  },
+  "ext_temp": {
+    "equation": "A-40",
+    "minValue": "-40",
+    "maxValue": "80",
+    "type": "Number",
+    "command": "22DD05",
+    "ecu": "7E2"
+  },
+  "batt_temp": {
+    "equation": "A-50",
+    "minValue": "-40",
+    "maxValue": "80",
+    "type": "Number",
+    "command": "224800",
+    "ecu": "7E4"
+  },
+  "is_charging": {
+    "equation": "A==2",
```

```suggestion
    "equation": "(A==2)||(A==8)",
```
This should contain both slow and fast charging

---

My 2012 Ford Focus Electric doesn't support DC charging :)